test/metabase/api/database_test.clj

```diff
@@ -83,7 +83,7 @@
    (merge
     (mt/object-defaults Database)
     (select-keys db [:created_at :id :details :updated_at :timezone :name :dbms_version
-                     :metadata_sync_schedule :cache_field_values_schedule])
+                     :metadata_sync_schedule :cache_field_values_schedule :uploads_enabled])
```

Comment:
Fix broken test that would fail if certain other tests were run that changed the value of `uploads_enabled`